custom_model_runner/datarobot_drum/drum/drum.py

```diff
@@ -1164,3 +1172,71 @@ def create_custom_inference_model_folder(code_dir, output_dir):
         fp.write(readme)
     if files_in_output & copied_files:
         print("Files were overwritten: {}".format(files_in_output & copied_files))
+
+
+def get_default_parameter_values(model_metadata: Dict) -> Dict[str, Union[int, float, str]]:
+    """Retrieve default parameter values from the hyperparameter section of model-metadata.yaml.
+    When `default` is provided, return the default value.
+    When `default` is not provided:
+        - Return `min` value if it is a numeric parameter.
+        - Return an emtpy string if it is a string parameter.
+        - Return the first allowed value if it is the select parameter.
+        - Return the default value of the first component parameter (sorted by parameter type) if it is a multi
+          parameter.
+    """
+
+    def _get_default_numeric_param_value(param_config: Dict) -> Union[int, float]:
+        """Get default value of numeric parameter."""
+        param_default_value = param_config.get("default")
+        if param_default_value is not None:
+            return param_default_value
+        else:
```

Comment:
style nit, I don't think there is a need for an else statement.

Reply:
Could you be more specific on this?

Reply:
since the `if` block returns, you can remove the else and it'll functionally be the same

Reply:
updated
https://github.com/datarobot/datarobot-user-models/blob/3f5e21b85c9a075401eb3cf13e13975c15648b97/custom_model_runner/datarobot_drum/drum/drum.py#L1178-L1199